playground/mocks/data/idp/idx/error-authenticator-enroll-phone-invalid-number.json

```diff
@@ -6,6 +6,60 @@
   "remediation":{
     "type":"array",
     "value":[
+      {
```

Comment:
This mock needed to be updated to now include the `currentAuthenticator` since that is returned in the response on errors too